andela/bp-esa-backend
# #160436783 Add users to channel

#### What does this PR do?
Enables users to be invited to channels.

#### Description of Task to be completed?
Placed Developers and SC-Manager if developer is an SC should be added to `p-partner` and `p-partner-int` slack channels.

#### How should this be manually tested?
Assuming you are an administrator of the workspace:

- Create an app. It should have workspace privileges in order to create channels(groups:write)
- Generate a token for the app and add it to the .env file as the SLACK_TOKEN
- Calling the `addToChannel` function in server/modules/slackIntegration.js, while providing the user's email and channel ID, will add user to channel.

#### What are the relevant pivotal tracker stories?
[160436783](https://www.pivotaltracker.com/story/show/160436783)
